### ISSUE TITLE
Scrolling with iPadOS Mouse.

### ISSUE THREAD

**Issue by WilsontheWolf** [novnc/noVNC#1505]
Recently, I have acquired  a new iPad and am using noVNC to use my pc. I am also using the magic keyboard. It works lovely and is the only (free) method I could find that works great with it. I was wondering if this could be implemented. The scrolling does work in thr browser but not noVNC. 

Here is a video which (attempts) to show it working in the browser but not the noVNC. Thank you so much for making this.

https://user-images.githubusercontent.com/33164598/103145071-632eb480-46f1-11eb-9b3a-49d459e7052c.MP4

**Comment by samhed** [comment]
Must be some special sort of scroll events that it sends then.. The regular magic mouse works fine on macOS. Do you have the possibility to plug in your iPad to a mac and check the browser console?

**Comment by WilsontheWolf** [comment]
Hey sorry for the late response. I don't have a mac available, but since I am using the chrome on iOS browser I can access the logs (but not send anything to the console). I could get access to a mac if you really needed to.

**Comment by WilsontheWolf** [comment]
Hey I'm using the magic keyboard's touchpad but I did hook up my logitech mouse at one point and it would scroll too.

Specifically this is actually a bug in webkit that was fixed in iPadOS 14.5, and I can confirm it works now fine. See this related issue on code-server https://github.com/cdr/code-server/issues/1455#issuecomment-828781910 and [the webkit](https://bugs.webkit.org/show_bug.cgi?id=210071) issue. 

https://user-images.githubusercontent.com/33164598/131265162-921cb72b-f120-4296-af33-6dcdd128600b.MP4
